src/main/java/com/theironyard/war/Game.java

```diff
@@ -0,0 +1,126 @@
+package com.theironyard.war;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+
+/**
+ * Created by Nigel on 6/24/16.
+ */
+public class Game {
+    Player player1;
+    Player player2;
+    public static final int TOPOFDECK = 0;
+    public Game(Player player1, Player player2){
+        this.player1 = player1;
+        this.player2 = player2;
+    }
+
+    /**
+     * Sets up the Deck and player's hands.
+     */
+    public void setupPlayers(){
+
+        //Setup Deck for players
+        Deck warDeck = Deck.makeStandardDeck();
+        warDeck.shuffle();
+        int deckSize = warDeck.getSize();
+
+        //Setup player hands.
+        System.out.format("There are %s Cards in the deck :)\n", deckSize);
+        while (warDeck.getSize() > 0){//for (int i = deckSize; i >= 0; i--){
+            //System.out.println(i);
+            player1.takeCard(warDeck.dealCard());
+            player2.takeCard(warDeck.dealCard());
```

Comment:
This line will explode when you have an odd number of cards.  This is due to the amount of cards only being check once per every 2 cards drawn.  One other way to do this would be to use a `current_player` like in the game of sticks.  Then you can draw a card and give it to the current player before switching players.  Additionally, you could move the code into the deck class and create a `splitDeck` call that will return 2 decks with as close to even card counts as possible.